UPGRADING.rst

```diff
@@ -18,7 +18,7 @@ This PR introduces two new deployment-specific environment variables,
 these variables from the example deployment to your personal deployment.
 
 Service accounts must be registered and authorized with SAM for integration
-tests to pass. See section 3.2.1 of the README for resgistration instructions.
+tests to pass. See section 3.2.1 of the README for registration instructions.
```

Comment:
The fix for `resgistration` shouldn't be in the commit it is currently in but in a fixup on an earlier commit.